ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -323,15 +322,34 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
     },
     updateVocabularyType: function(e) {
       this.setState({vocabularyType: e.target.value});
-      e.target.isChecked = true;
     },
     updateMultiTerms: function(e) {
       this.setState({multiTerms: e.target.checked});
     },
     submitForm: function(e) {
-      var API_ROOT_VOCAB_URL = '/api/v1/repositories/' + this.props.repoSlug +
-        '/vocabularies/';
       e.preventDefault();
+      if (_.isEmpty(this.state.name)) {
```

Comment:
It should never do that, any number in a slug field or in a url should be treated as just another slug. If you changed the name to a number it would look as if the vocabulary was looked up by id.